### PR TITLE
add last_login_dt and make email non-mandatory and updated apis

### PR DESCRIPTION
**Pull request by Vihit**

### CODE REVIEW

File: logever/src/main/java/ai/smartfac/logever/entity/User.java [hunk 1]
```diff
@@ -45,11 +47,16 @@ public class User implements UserDetails {
...
+    @JsonIgnore
     private String updatedBy;
     @JsonIgnore
     @Column(name="update_dt")
     @UpdateTimestamp
     private Timestamp updateDt;
+    @Column(name = "is_active",nullable = false,columnDefinition = "BOOLEAN")
+    private Boolean isActive;
+    @Column(name = "last_login_dt")
+    private Timestamp lastLoginDt;
```
Comment (cpandya231) [inline logever/src/main/java/ai/smartfac/logever/entity/User.java:59]: along with this, we will also need created_at.

**Review by cpandya231 — CHANGES_REQUESTED** [review]

File: logever/src/main/java/ai/smartfac/logever/entity/User.java (hunk 1 above)
Comment (cpandya231) [reply logever/src/main/java/ai/smartfac/logever/entity/User.java:59]: Disregard, found createDt.

**Review by cpandya231 — APPROVED** [review]

File: logever/src/main/java/ai/smartfac/logever/entity/User.java [hunk 2]
```diff
@@ -45,11 +47,16 @@ public class User implements UserDetails {
     private Timestamp createDt;
     @Column(name = "updated_by")
     @LastModifiedBy
+    @JsonIgnore
     private String updatedBy;
     @JsonIgnore
     @Column(name="update_dt")
     @UpdateTimestamp
     private Timestamp updateDt;
```
Comment (cpandya231) [inline logever/src/main/java/ai/smartfac/logever/entity/User.java:55]: We will also need status of the user, ACTIVE/INACTIVE

**Review by cpandya231 — CHANGES_REQUESTED** [review]

File: logever/src/main/java/ai/smartfac/logever/entity/User.java (hunk 2 above)
Comment (Vihit) [reply logever/src/main/java/ai/smartfac/logever/entity/User.java:54]: we have isActive.

File: logever/src/main/java/ai/smartfac/logever/entity/User.java (hunk 1 above)
Comment (Vihit) [reply logever/src/main/java/ai/smartfac/logever/entity/User.java:58]: removed jsonignore and createDt is now available in response.

**Review by cpandya231 — APPROVED** [review]

**Pull request merged**